Android/app/src/main/res/values/strings.xml

```diff
@@ -9,6 +9,7 @@
     <string name="navigation_drawer_travel">Travel</string>
     <string name="navigation_drawer_utilities">Utilities</string>
     <string name="navigation_drawer_medals">Medals</string>
+    <string name="navigation_drawer_about_us">About Us</string>
     <string name="navigation_drawer_signout">Sign Out</string>
     <string name="navigation_drawer_nav_report_bug">Report A Bug</string>
```

Comment:
Remove this string

Reply:
Then how should I write "About Us"?

Reply:
I meant to remove `navigation_drawer_nav_report_bug` string